packages/interfaces/src/protocols.ts

```diff
@@ -87,6 +88,10 @@ export type ProtocolCreateOptions = {
    * Use recommended bootstrap method to discovery and connect to new nodes.
    */
   defaultBootstrap?: boolean;
+  /**
+   * List of peers to use to bootstrap the node. Ignored if defaultBootstrap is turned on.
+   */
+  bootstrapPeers?: MultiaddrInput[];
```

Comment:
let's not ignore if `defaultBootstrap` is `true`. 
let's use this nonetheless as well